wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -503,6 +583,404 @@ Disable TLS (Not Recommended)
+
+```
+curl -X POST \
+  https://localhost:8090/api/v1/Ae2tdPwUPE...8V3AVTnqGZ/accounts \
+  -H 'Content-Type: application/json;charset=utf-8' \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "name": "MyOtherAccount",
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
+}'
+```
+
+Note that the `spendingPassword` here should match the one provided earlier in [Creating a
+New Wallet](#section/Getting-Started/Creating-a-New-Wallet).
+
+
+```json
+{
+    "status": "success",
+    "data": {
+        "index": 2902829384,
+        "addresses": [
+            "DdzFFzCqrh...Sj5a2rkx8y"
+        ],
+        "amount": 0,
+        "name": "MyOtherAccount",
+        "walletId": "Ae2tdPwUPE...8V3AVTnqGZ",
+    },
+    "meta": { ... }
+}
+```
+
+You can always retrieve this account description later if needed via [`GET /api/v1/wallets/{{walletId}}/accounts/{{accountId}}`](#tag/Accounts%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts~1{accountId}%2Fget).
+
+For example:
+
+```
+curl -X GET \
+  https://127.0.0.1:8090/api/v1/wallets/Ae2tdPwUPE...8V3AVTnqGZ/accounts/2902829384 \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt
+```
+
+For a broader view, the full list of accounts of a given wallet can be retrieved using [`GET /api/v1/wallets/{{walletId}}/accounts`](#tag/Accounts%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts%2Fget)
+```
+curl -X GET \
+  https://127.0.0.1:8090/api/v1/wallets/Ae2tdPwUPE...8V3AVTnqGZ/accounts \
+  -H 'Accept: application/json;charset=utf-8' \
+  --cacert ./scripts/tls-files/ca.crt
+```
+
+```json
+{
+    "status": "success",
+    "data": [
+        {
+            "index": 2147483648,
+            "addresses": [
+                "DdzFFzCqrh...U6AGwxWmw"
+            ],
+            "amount": 0,
+            "name": "Initial account",
+            "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+        },
+        {
+            "index": 2902829384,
+            "addresses": [
+                "DdzFFzCqrh...Sj5a2rkx8y"
+            ],
+            "amount": 0,
+            "name": "MyOtherAccount",
+            "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+        }
+    ],
+    "meta": {
+        "pagination": {
+            "totalPages": 1,
+            "page": 1,
+            "perPage": 10,
+            "totalEntries": 2
+        }
+    }
+}
+```
+
+
+Managing Addresses
+------------------
+
+By default, wallets you create are provided with an account which has one default address. It
+is possible (and recommended) for an account to manage multiple addresses. Address reuse
+actually reduces privacy for it tights more transactions to a small set of addresses.
+
+Note that when paying, this is fairly abstracted from you: addresses are selected from an
+account of your wallet based on different strategies and policies.
```

Comment:
I'd reword:

> When paying, the wallet makes many of these choices for you. Addresses are selected from a wallet's account based on several different strategies and policies.

Is there a way to point to these strategies and policies, or give a user a way of knowing/understanding what they are or how to have any control over them?

Reply:
Not sure whether we have any public document on the topic but we could drop a line about those policies? Maybe as a use-case section "Getting More Control Over the Wallet's strategies"

Reply:
Good point -- historically we introduced these policies to support different use cases, as typically exchanges are interested in different tradeoffs than normal users. This is an interesting article on the topic: https://medium.com/@lopp/the-challenges-of-optimizing-unspent-output-selection-a3e5d05d13ef

Said that, input selection and the policies are going to be reworked in the upcoming data layer, so really the _single_ take-away is that "normal users" shouldn't bother, but exchanges **must** use the `HighThroughput` policy, or they would get too-big transactions. That is definitely worth clarifying :)